deptrac.yaml

```diff
@@ -238,6 +239,7 @@ deptrac:
       - Support
     # Legacy Database
     LegacyDatabase:
+      - LegacyCommunity
```

Comment:
This reads like a circular reference.

Reply:
True. It allows both layers to talk to each other, which technically is bad design. Not much of an issue in this case though.

The legacy database layer may talk to its community layer while the community layer may also define stuff in the database layer.